filelink.go

```diff
@@ -1,14 +1,27 @@
+//
+//
+// File generated from our OpenAPI spec
+//
+//
+
 package stripe
 
-import (
-	"encoding/json"
-)
+import "encoding/json"
+import "github.com/stripe/stripe-go/v72/form"
 
 // FileLinkParams is the set of parameters that can be used when creating or updating a file link.
 type FileLinkParams struct {
-	Params    `form:"*"`
-	ExpiresAt *int64  `form:"expires_at"`
-	File      *string `form:"file"`
+	Params       `form:"*"`
+	ExpiresAt    *int64  `form:"expires_at"`
+	ExpiresAtNow *bool   `form:"-"` // See custom AppendTo
+	File         *string `form:"file"`
+}
+
+// AppendTo implements custom encoding logic for FileLinkParams.
+func (f *FileLinkParams) AppendTo(body *form.Values, keyParts []string) {
+	if BoolValue(f.ExpiresAtNow) {
+		body.Add(form.FormatKey(append(keyParts, "ExpiresAt")), "now")
```

Comment:
This should be `body.Add(form.FormatKey(append(keyParts, "expires_at")), "now")` right? It jumped out at me but now I'm worried I missed some in previous reviews.
We likely need to add custom tests for all custom `AppendTo` to avoid this

Reply:
**Excellent** catch, slipped right by me. This is the first `AppendTo` I've tried to codegen so nothing slipped past I don't think.

Reply:
(this is fixed)